projects/flyout/src/modules/flyout/flyout.component.html

```diff
@@ -1,9 +1,13 @@
 <div
   class="sky-flyout"
   tabindex="-1"
-  [attr.role]="config?.ariaRole"
+  [attr.role]="config?.ariaRole ? config.ariaRole : 'dialog'"
```

Comment:
My understanding from our flex time was no. We have the option so we can't remove it until Sky UX 6 - and removing it or ignoring it would be a breaking change. While we can consider this an accessibility bug - we felt that we should still respect what people have put in for now. We added the deprecation warning to the option and put in the default. I also only set `aria-modal` if the role is `dialog` via the consumer or the default so that we weren't breaking existing implementations more.